Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "nydus-rs"
 version = "2.0.0-rc.0"
 authors = ["The Nydus Developers"]
 edition = "2018"
+license = "Apache-2.0 OR BSD-3-Clause"
```

Comment:
The BSD-3-Clause need special treatment, and seems we have only Apache 2.0 license file attached.